packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -101,6 +101,20 @@ export class VaMemorableDate {
     // will happen afterwards
     validate(this, yearNum, monthNum, dayNum);
     this.dateBlur.emit(event);
+
+    if (this.enableAnalytics) {
+      const detail = {
+        componentName: 'va-memorable-date',
+        action: 'blur',
+        details: {
+          label: this.label,
```

Comment:
And here it's just `label` instead of `date-blur-label` or something like that.

Reply:
I've updated this to just `label` because of the way the vets-website script is transforming it for the datalayer.